Train = True. Doodle list: ['sketchy_doodle', 'tuberlin', 'google_doodles'],
 real list: ['sketchy_real', 'google_real', 'cifar'].
 classes: dict_keys(['airplane', 'car', 'cat', 'dog', 'frog', 'horse', 'truck', 'bird', 'ship'])
Doodle data size 7022, real data size 46364, ratio 0.15145371408851696
Train = False. Doodle list: ['sketchy_doodle', 'tuberlin', 'google_doodles'],
 real list: ['sketchy_real', 'google_real', 'cifar'].
 classes: dict_keys(['airplane', 'car', 'cat', 'dog', 'frog', 'horse', 'truck', 'bird', 'ship'])
Doodle data size 1764, real data size 9341, ratio 0.18884487742211756
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x7f624b984290>
Traceback (most recent call last):
  File "/home/rishabh/miniconda3/envs/Rish4243/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1358, in __del__
    self._shutdown_workers()
  File "/home/rishabh/miniconda3/envs/Rish4243/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1322, in _shutdown_workers
    w.join(timeout=_utils.MP_STATUS_CHECK_INTERVAL)
  File "/home/rishabh/miniconda3/envs/Rish4243/lib/python3.7/multiprocessing/process.py", line 140, in join
    res = self._popen.wait(timeout)
  File "/home/rishabh/miniconda3/envs/Rish4243/lib/python3.7/multiprocessing/popen_fork.py", line 45, in wait
    if not wait([self.sentinel], timeout):
  File "/home/rishabh/miniconda3/envs/Rish4243/lib/python3.7/multiprocessing/connection.py", line 921, in wait
    ready = selector.select(timeout)
  File "/home/rishabh/miniconda3/envs/Rish4243/lib/python3.7/selectors.py", line 415, in select
    fd_event_list = self._selector.poll(timeout)
KeyboardInterrupt: